packages/extension-chakra-store-locator/README.md

```diff
@@ -43,7 +43,7 @@ The Store Locator extension is configured via the `mobify.app.extensions` proper
     "app": {
       "extensions": [
         [
-          "@salesforce/extension-store-locator",
+          "@salesforce/extension-chakra-store-locator",
```

Comment:
Near the bottom of this readme, there are file paths that need to be updated too.